packages/app-utils/src/components/ConnectionBootstrap.tsx

```diff
@@ -30,7 +30,7 @@ export function ConnectionBootstrap({
   const api = useApi();
   const client = useClient();
   const [error, setError] = useState<unknown>();
-  const [connection, setConnection] = useState<IdeConnection>();
+  const [connection, setConnection] = useState<dh.IdeConnection>();
```

Comment:
A fun PR to watch about importing classes from namespaced packages: https://github.com/Microsoft/TypeScript/issues/13135